config/initializers/elasticsearch.rb

```diff
@@ -0,0 +1,7 @@
+require "elasticsearch"
+
+Search = Elasticsearch::Client.new(
```

Comment:
Since we explicitly install the dependency, I think it makes sense to specify `adapter: :typhoeus` to the options instead of relying on the private ` __auto_detect_adapter` helper method in `Elasticsearch::Transport`.

Reply:
good call. looks like the test suite needs a little attention as well